### PR TITLE
fix: handle empty event definitions

### PR DESCRIPTION
**Pull request by pinussilvestrus**
__Which issue does this PR address?__

This prevents the plugin from crashing if `eventDefinitions` are an empty array. We can not guarantee it will always be undefined, especially if we or plugins are using `businessObject.get('eventDefinitions')`.

Related to https://github.com/camunda/camunda-modeler-plugins/issues/35

__Acceptance Criteria__

* [ ] Corresponds to the concept 
* [ ] Corresponds to the design 

__Definition of Done__

* [ ] corresponds to [the design principles](https://github.com/bpmn-io/design-principles)
* [ ] corresponds to [the code standards](https://github.com/bpmn-io/.github/blob/master/.github/CONTRIBUTING.md#create-a-pull-request)
* [ ] passes Continuous Integration checks
* [ ] available as feature branch on GitHub
  * [ ] contains the cleaned up commit history
  * [ ] commit messages satisfy our [commit message guidelines](https://www.conventionalcommits.org/)
  * [ ] a single commit closes the issue via Closes #issuenr

### CODE REVIEW

**Comment by pinussilvestrus** [comment]
I'll put a GitHub actions migration on top. 🚧

File: test/spec/TransactionBoundariesSpec.js [hunk 1]
```diff
@@ -142,6 +142,21 @@ describe('transaction-boundaries', function() {
 
     }));
 
+
+    it('should not crash on empty event definitions', inject(function(transactionBoundaries, elementRegistry) {
```
Comment (MaxTru) [inline test/spec/TransactionBoundariesSpec.js:146]:
```suggestion
    it('should not crash on event definitions being an empty Array', inject(function(transactionBoundaries, elementRegistry) {
```

**Review by MaxTru — COMMENTED** [review]
1 minor comment and 1 requested change: `npm audit` yields 12 high severity vulnerabilities, but they can be fixed without attention using `npm audit fix`. Any reason not to do that?

File: test/spec/TransactionBoundariesSpec.js (hunk 1 above)
Comment (pinussilvestrus) [reply test/spec/TransactionBoundariesSpec.js]: Thanks 👍

**Comment by pinussilvestrus** [comment]
Thanks for your suggestions @MaxTru 👍  I took the audit errors as an occasion to update the dependencies at all, including updating the test suite to bpmn-js@8 (Promises).

**Review by MaxTru — APPROVED** [review]
Nice house cleaning! :broom: :house:

**Comment by pinussilvestrus** [comment]
Good old days

image

**Pull request merged**